src/main/java/com/xebia/incubator/xebium/SeleniumDriverFixture.java

```diff
@@ -505,6 +648,17 @@ private String executeCommand(final ExtendedSeleniumCommand command, final Strin
 			return null;
 		}
 
+		if ("contextMenu".equals(command.getSeleniumCommand())) {
```

Comment:
`executeCommand` is getting increasingly complicated - could we perhaps model this as a `SeleneseCommand<Void>` (kind of like `Type`)?